packages/rocketchat-api/server/api.js

```diff
@@ -325,6 +325,22 @@ class API extends Restivus {
 			}
 			return response;
 		};
+		const forgotPassword = function() {
```

Comment:
Honestly, I'm not sure if this should live in here or in the users file. Thoughts @sampaiodiego

Reply:
I agree, the endpoint could be something like `/users.forgotPassword`

Reply:
My thoughts as well. :+1: